README.md

```diff
@@ -36,17 +36,21 @@ struct MyCheesyStruct {
 }
 ```
 
-A new trait will be produced called `MyCheesyStructGetters`. This trait will exist in the same module as `MyCheesyStruct`.
+A new impl will be produced for `MyCheesyStruct`.
 ```rust
-pub trait MyCheesyStructGetters {
-    fn get_x(&self) -> i64;
-    fn get_y(&self) -> i64;
+impl MyCheesyStruct {
+    fn get_x(&self) -> i64 {
+        &self.x
+    }
+
+    fn get_y(&self) -> i64 {
```

Comment:
I guess you forgot to change the function names here?

Reply:
It's fixed. Try 0.0.7.

Reply:
I've fixed it.